redpanda/Chart.yaml

```diff
@@ -20,7 +20,7 @@ maintainers:
   - name: redpanda-data
     url: https://github.com/orgs/redpanda-data/people
 type: application
-version: 2.2.1
+version: 2.2.2
```

Comment:
nit: we're at 22.2.6 if you're updating this

Reply:
This is the chart version. See appVersion for the version of Redpanda.